projects/cm_mcu/LocalTasks.c

```diff
@@ -583,21 +585,21 @@ int8_t getFFtemp(const uint8_t i)
   configASSERT(i < NFIREFLIES);
   if (i < NFIREFLIES_IT_F1) {
     int index = i * (ffl12_f1_args.n_commands * ffl12_f1_args.n_pages) + i1;
-    val = ffl12_f1_args.sm_values[index];
+    val = (int8_t)ffl12_f1_args.sm_values[index];
   }
 
   else if (NFIREFLIES_IT_F1 <= i && i < NFIREFLIES_IT_F1 + NFIREFLIES_DAQ_F1) {
     int index = (i - NFIREFLIES_IT_F1) * (ffldaq_f1_args.n_commands * ffldaq_f1_args.n_pages) + i1;
-    val = ffldaq_f1_args.sm_values[index];
+    val = (int8_t)ffldaq_f1_args.sm_values[index];
```

Comment:
will have to check if it's (int16_t) or (int8_t).

Reply:
The value printed out and the return value of the function is `int8_t` so that is why I used that for the cast.

Reply:
Okay, I see your point. I was looking at the `sm_values`'s type.